blm_cluster.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 
+RealPath() {
+    (cd $(dirname "$1") && pwd)
+}
+
+BLM_PATH=$(RealPath "${BASH_SOURCE[0]}")
+
 # include parse_yaml function
-. lib/parse_yaml.sh
+. $BLM_PATH/lib/parse_yaml.sh
 
 # Work out if we have been given multiple analyses configurations
 # Else just assume blm_config is the correct configuration
 if [ "$1" == "" ] ; then
-  cfg=$(realpath "blm_config.yml")
+  cfg=$(RealPath "blm_config.yml")
```

Comment:
Hi @nicholst , I am finding that this command is only giving the directory. E.g. It gives;

`/users/nichols/inf852/BLM-py`

Rather than 

`/users/nichols/inf852/BLM-py/blm_config.yml`